### PR TITLE
Store form user data local storage

### PR DESCRIPTION
**Pull request by abdallahmalima**
**Note:**'I have been made solo late in the week with approval from Student Success. Please reach out to Marco Lizza in the Student Success team if you need to verify it.'

image

**In this part I added the following functionality:**

-   When the user changes the content of any input field, the data is saved to the local storage.
-   When the user loads the page if there is any data in the local storage the input fields are pre-filled with this data.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -270,4 +275,52 @@ contact_form.addEventListener('submit',(event)=>{
...
+    localStorage.setItem('user',JSON.stringify(user));
+    user=JSON.parse(localStorage.getItem('user'));
+    email_el.value=user.email;
+  });
+
+  message_el.addEventListener('change',()=>{
+    user.message=message_el.value;
+    localStorage.setItem('user',JSON.stringify(user));
+    user=JSON.parse(localStorage.getItem('user'));
+    message_el.value=user.message;
```
Comment (cwaku) [inline app.js]: [OPTIONAL]
- [ ] You happen to be violating the DRY principle of [clean code](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices). I recommend that you find a better way to achieve this that doesn't require you to repeat logic. For instance, you could get all the `input` fields from the `form` using the `document.querySelectorAll` when the `DOM` is loaded, which will return them in an array. From there, you could use [array iteration methods](https://www.w3schools.com/js/js_array_iteration.asp) to get the values for each and then save it to the `user` object.

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @abdallahmalima 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

- [ ] Good job setting up linters to catch errors and promote consistency in your codebase. However, you only have the checks for HTML and CSS, not JavaScript. I recommend that you change the configuration file from [html&css](https://github.com/microverseinc/linters-config/blob/master/html-css) to [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) (for projects that require you to use both JavaScript and HTML & CSS)
- [ ] To make your repo more professional, I suggest that you remove all section you have not provided information for and give a proper description of "Getting Started" section
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -10,31 +10,25 @@
...
-### Install
-
+Clone the repository
 ### Usage
-
-### Run tests
-
-### Deployment
-
+Open index.html file
```
Comment (rloterh) [inline README.md:32]: - [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide instructions below it.
Implementing the above change(s) will make your ReadMe stand out even better 😉

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Abdallah,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Data is persisted correctly to the local storage

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_

keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**